cli.rb

```diff
@@ -0,0 +1,32 @@
+class CLI
+  attr_accessor :current_attempts
+
+  def initialize(current_attempts = 0)
```

Comment:
Why did you move @current_attempts to cli. It is better to have it inside main.rb

Reply:
Fixed